components/script/dom/dompointreadonly.rs

```diff
@@ -112,6 +115,37 @@ impl DOMPointReadOnlyMethods<crate::DomTypeHolder> for DOMPointReadOnly {
     fn W(&self) -> f64 {
         self.w.get()
     }
+
+    // https://dev.w3.org/fxtf/geometry/Overview.html#dom-dompointreadonly-matrixtransform
+    // https://drafts.fxtf.org/geometry/Overview.html#transform-a-point-with-a-matrix
+    fn MatrixTransform(
+        &self,
+        matrix: &DOMMatrixInit,
+        can_gc: CanGc,
+    ) -> Fallible<DomRoot<DOMPoint>> {
+        let matrix_object = match dommatrixinit_to_matrix(matrix) {
+            Ok(converted) => converted,
+            Err(exception) => {
+                return Err(exception);
+            },
+        };
+        let x = self.X();
```

Comment:
better to add spec steps, I knew old code not doing that but better to do i tin new code:

```suggestion
      // Let x be point’s [x coordinate](https://drafts.fxtf.org/geometry/Overview.html#point-x-coordinate).
        let x = self.X();
```